tests/handlebars/whitespace/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -407,6 +407,33 @@ sometimes{{nogaps}}areimportant<Hello></Hello>
 
 </div>
 
+<div class="attr
```

Comment:
There is a `tests/handlebars/attr-node` test folder. Maybe we should create an `attributes.hbs` files there?

Reply:
Good idea, I'll move it there.